lib/phoenix_live_view.ex

```diff
@@ -217,6 +217,13 @@ defmodule Phoenix.LiveView do
       this option will override any layout previously set via
       `Phoenix.LiveView.Router.live_session/2` or on `use Phoenix.LiveView`
 
+    * `:auto_connect` - if false, instructs the LiveView JavaScript client
+      to not automatically connect to the server on dead render.
```

Comment:
AFAIK we don't use "dead" in the docs (and we should avoid doing so, IMO)

```suggestion
      to not automatically connect to the server on disconnected render.
```